File: utilities/src/test/java/org/gradle/sample/utilities/JoinUtilsTest.java
package org.gradle.sample.utilities;

import org.gradle.sample.list.LinkedList;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class JoinUtilsTest {

    @Test
    public void testJoin() {
        LinkedList list = new LinkedList();
        list.add("one");
        list.add("two");
        var result = JoinUtils.join(list);
        assertEquals("one two", result);
    }

}

File: utilities/src/test/kotlin/org/gradle/sample/utilities/JoinUtilsTest.kt
package org.gradle.sample.utilities

import org.gradle.sample.list.LinkedList
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class JoinUtilsTest {
    @Test
    fun testJoin() {
        val list = LinkedList()
        list.add("one")
        list.add("two")
        val result = JoinUtils.join(list)
        Assertions.assertEquals("one two", result)
    }
}